### DISCUSSION TITLE
I can not upload sitemap.xml (error Not Found)

### DISCUSSION THREAD

**Discussion by alexander-sml** [requarks/wiki#4169] [General]
I tried to download by url: example.com/sitemap.xml, but I got: Not Found.
I found a thread about [it](https://feedback.js.wiki/wiki/p/automatically-generate-a-sitemap-for-the-wiki), unfortunately it does not work for me.
Could you provide me: it is bug or specific of work?
I use the latest version of Wiki JS - 2.5.201

**Comment by Max2983** [comment]
Have you allowed robots and indexation in General -> SEO settings?

**Reply by alexander-sml** [reply]
Yes, I have
![Screenshot_183](https://user-images.githubusercontent.com/5771926/166472731-75ee043c-2182-428c-9b54-5996525a82e3.png)

**Comment by Philipinho** [comment]
I have just released a sitemap too for Wiki.js. You can check it out https://github.com/hostwiki/wikijs-sitemap
